### DISCUSSION TITLE
Add option to put read-only on directory mount

### DISCUSSION THREAD

**Discussion by jlucaso1** [coollabsio/coolify#3422] [Feature Requests]
![image](https://github.com/user-attachments/assets/6e3b1721-9f5b-44aa-ab98-22e2074884b5)

**Comment by neverabsolute** [comment]
+1, this would be very useful

**Comment by H01001000** [comment]
Looking at the code I found you can add ":ro" suffix at mount path, since Coolify doesn't validate the mount path and it just concat
see https://github.com/coollabsio/coolify/blob/b7282ad565376ac3efcc70c8ca383b765cc79b2b/app/Jobs/ApplicationDeploymentJob.php#L2633
